src/main/java/uk/co/codezen/maven/redlinerpm/rpm/RpmPackage.java

```diff
@@ -288,6 +293,26 @@ public String getName()
     }
 
     /**
+     * Set override name
+     *
+     * @param overrideName override name
+     */
+    public void setOverrideName(String overrideName)
```

Comment:
I'd name it `nameOverride`. `overrideName` sounds like a boolean flag to me. I'd prefer seeing
```<nameOverride>foo</nameOverride>``` than ```<overrideName>foo</overrideName>```